README.md

```diff
@@ -65,6 +65,35 @@ At the moment, no live-reload of certificates is supported, so to use new certif
 The TLS endpoint also supports TLS client authentication, which can be used to create sophisticated authentication and authorization structures easily.
 The intention is to have the TLS endpoint exposed to the inner application (your **backend**) and to have the WS endpoint exposed to your end users via a reverse proxy like nginx (which also serves the files for your **frontend**) which further separates TLS termination away from `autobahnkreuz` to your cloud provider/environment. The separation between TLS for internal usage and WS for external usage also yields a better privilege separation and therefore improves application security.
 
+## Logging Configuration
+Configuration of the `autobahnkreuz` Logger is supported by setting following environment variables:
+
+`SERVICE_LOGFORMAT`
```

Comment:
You also want to structure this better.

Reply:
Changed var names to subsections to be navigable in section trees.